components/src/core/dropdown-toolbar/dropdown-toolbar.component.scss

```diff
@@ -21,7 +21,7 @@
 
 .pxb-dropdown-toolbar-text-content-container {
     display: block;
-    padding-right: 16px;
+    max-width: 100%;
```

Comment:
Is 100% too big here if there is an icon? Do we need a max-width set?

Reply:
Yeah, it does get a little weird with an icon. Maybe 50%?

Reply:
do we need a max-width at all? What's wrong with auto?

Reply:
You can take the max-width off and apply `overflow: hidden`, that will give the behavior you want